brd-ios/breadwallet/src/Redesign/Scenes/Buy/OrderPreview/Views/BuyOrderView.swift

```diff
@@ -208,12 +234,25 @@ class BuyOrderView: FEView<BuyOrderConfiguration, BuyOrderViewModel> {
         titleLabel.setup(with: viewModel?.title)
         currencyIconImageView.setup(with: viewModel?.currencyIcon)
         currencyNameLabel.setup(with: viewModel?.currencyAmountName)
+        
         rateView.setup(with: viewModel?.rate)
+        rateView.isHidden = viewModel?.rate == nil
+        
+        [titleLabel, currencyStackView, rateView, topLineView].forEach { view in
```

Comment:
do all need 2 be set hidden? they dont have a common superview, that could hide all 4?

Reply:
Nope, we don't have anything common there thanks to the designs. Maybe in our new redesign...